README.md

```diff
@@ -98,4 +104,17 @@ go get -u github.com/sourcegraph/src-cli/cmd/src
 3.  Travis will automatically perform the release. Once it has finished, **confirm that the curl commands fetch the latest version above**.
 4.  Update the `MinimumVersion` constant in the [src-cli package](https://github.com/sourcegraph/sourcegraph/tree/master/internal/src-cli/consts.go).
 
-**Note**: The version recommended by a Sourcegraph instance will be the highest patch version with the same major and minor version as the set minimum. This means that patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of src-cli without having to update the instance.
+### Patch releases
+
+The version recommended by a Sourcegraph instance will be the highest patch version with the same major and minor version as the set minimum. This means that patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of src-cli without having to update the instance.
+
+If you are releasing a bug fix or a new feature that is backwards compatible with one of the previous two minor version of Sourcegraph, the changes should be cherry-picked into a patch branch and re-releases with a new patch version. For example, suppose we have the the recommended versions.
+
+| Sourcegraph version | Recommended src-cli version |
+| ------------------- | --------------------------- | 
+| `3.x`               | `3.y.z`                     |
+| `3.{x-1}`           | `3.{y-1}.w`                 |
+
+If a new feature is added to a new `3.y.{z+1}` release of src-cli and this change requires only features available in Sourcegraph `3.{x-1}`, then this feature should also be present in a new `3.{y-1}.{w+1}` release of src-cli. Similar logic should follow for Sourcegraph `3.{x-2}` and its recommended version of src-cli.
+
+Because a Sourcegraph instance will automatically select the highest patch version, almost all non-breaking changes should increment only the patch version. The process above only necessary if a backwards-compatible change is made _after_ a backwards-incompatible one. If the recommended src-cli version for Sourcegraph `3.{x-1}` was instead `3.y.x` in the example above, there is no additional step required, and the new patch version of src-cli will be available to both Sourcegraph versions.
```

Comment:
Is the `x` in `3.y.x` intended to match the `x` in `3.x` under the "Sourcegraph version" column?

Reply:
Updated.